src/components/FileUpload.vue

```diff
@@ -19,7 +19,7 @@
       </el-col>
     </el-row>
     <el-row>
-      <el-col v-if="state._file.state">
+      <el-col v-if="theFile.state">
```

Comment:
I think calling it `datasetFile` or something in that vein will be a bit more descriptive.

Reply:
`seletecedFile`?

Reply:
I think datasetFile is a bit more specific in the context (since it is a file with the dataset that the user want to upload to our system). Either works for me in this context though. So whatever you feel is more appropriate.